aws/resource_aws_vpc_endpoint_service.go

```diff
@@ -60,7 +60,7 @@ func resourceAwsVpcEndpointService() *schema.Resource {
 			},
 			"private_dns_name": {
 				Type:     schema.TypeString,
-				Computed: true,
```

Comment:
Leave `Computed: true` here as the value is computed if no explicit DNS name is specified.